components/event-overview-list/EventOverviewList.vue

```diff
@@ -0,0 +1,414 @@
+              :href="event.url"
+              class="event-overview-list__link"
+            >
+              {{ event.urlLabel }}
+            </a>
+            <nuxt-link
+              v-else
+              :to="`/en/events/${event.slug}`"
+              class="event-overview-list__link event-overview-list__link--internal"
+            >
+              Go to event page
+            </nuxt-link>
+          </div>
+
+          <lazy-media class="event-overview-list__list-item-image">
+            <img
+              :src="`${event.image.url}?auto=compress&w=600`"
+              alt=""
+            >
+          </lazy-media>
+
+          <div class="event-overview-list__list-item-footer">
+            <p class="event-overview-list__date">
+              {{ event.displayDate }}
+            </p>
+
+            <p class="event-overview-list__location">
+              {{ event.location }}
+            </p>
+          </div>
+        </li>
+      </ol>
+    </article>
+
+    <article
+      id="article2"
+      role="tabpanel"
+      aria-labelledby="tab2"
+      :hidden="currentTabId != 'tab2'"
+    >
+      <ol class="event-overview-list__list">
+        <li
+          v-for="event in pastEvents"
+          :key="event.id"
+          class="event-overview-list__list-item"
+        >
+          <div class="event-overview-list__list-item-header">
+            <h3
+              v-html="event.name"
+              class="event-overview-list__name"
+            />
+
+            <p class="event-overview-list__summary">
+              {{ event.summary }}
+            </p>
+
+            <a
+              v-if="event.url"
+              :href="event.url"
+              class="event-overview-list__link"
+            >
+              {{ event.urlLabel }}
+            </a>
+            <nuxt-link
+              v-else
+              :to="`/en/events/${event.slug}`"
+              class="event-overview-list__link event-overview-list__link--internal"
+            >
+              Go to event page
+            </nuxt-link>
+          </div>
+
+          <lazy-media class="event-overview-list__list-item-image">
+            <img
+              :src="`${event.image.url}?auto=compress&w=600`"
+              alt=""
+            >
+          </lazy-media>
+
+          <div class="event-overview-list__list-item-footer">
+            <p class="event-overview-list__date">
+              {{ event.displayDate }}
+            </p>
+
+            <p class="event-overview-list__location">
+              {{ event.location }}
+            </p>
+          </div>
+        </li>
+      </ol>
+    </article>
+  </section>
+</template>
+
+<script>
+  import LazyMedia from '~/components/lazy-media/LazyMedia';
+
+  export default {
+    props: {
+      internalEvents: Array,
+      externalEvents: Array,
+    },
+    components: { LazyMedia },
+    data () {
+      return {
+        currentTabId: 'tab1',
+        futureEvents: null,
+        pastEvents: null,
+      };
+    },
```

Comment:
**suggestion**: use computed props:
```js
computed: {
   today() { return new Date() },
   allEvents() { return [...this.internalEvents, ...this.externalEvents] },
   futureEvents() { 
     return this.allEvents
        .filter(...)
        .sort(...)
  },
   pastEvents() { 
     return this.allEvents
        .filter(...)
        .sort(...)
  }
}
```
Then force the computed properties to update on `mounted` by resetting `today`:
```js
mounted() {
  this.today = new Date();
}
```